activesupport/lib/active_support/notifications/instrumenter.rb

```diff
@@ -57,7 +57,8 @@ class Event
 
       def self.clock_gettime_supported? # :nodoc:
         defined?(Process::CLOCK_PROCESS_CPUTIME_ID) &&
-          !Gem.win_platform?
+          !Gem.win_platform? &&
+          RUBY_PLATFORM !~ /solaris/i
```

Comment:
How about using faster `match?`?

```suggestion
          !/solaris/i.match?(RUBY_PLATFORM)
```

Reply:
Good idea. I have done that now.